packages/terra-modal-manager/src/ModalManager.jsx

```diff
@@ -2,115 +2,31 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import Modal from 'terra-modal';
-
 import AppDelegate from 'terra-app-delegate';
 import SlideGroup from 'terra-slide-group';
 import breakpoints from 'terra-responsive-element/lib/breakpoints.scss';
+import DisclosureManager, { availableDisclosureSizes } from 'terra-disclosure-manager';
```

Comment:
With the new directory structure of examples being inside of the component package directory, you'll need to include components you use in your examples / tests as dependencies.

For example, the linter is complaining about:
```
  4:19  error  Unable to resolve path to module 'terra-modal'               import/no-unresolved
  4:19  error  Missing file extension for "terra-modal"                     import/extensions
  8:61  error  Unable to resolve path to module 'terra-disclosure-manager'  import/no-unresolved
  8:61  error  Missing file extension for "terra-disclosure-manager"        import/extensions
```

To resolve, add `terra-modal` and `terra-disclosure-manager`  as dependencies to this component.

Reply:
Sorry for the confusion, I'll update these. Since the Disclosure Manager hasn't been merged yet the install fails regardless so I hadn't gone back and updated these.